tests/playbooks/tests_wireless_wpa3_sae.yml

```diff
@@ -0,0 +1,44 @@
+# SPDX-License-Identifier: BSD-3-Clause
+---
+- hosts: all
+  vars:
+    interface: wlan0
+  tasks:
+    - name: "INIT: wireless tests"
+      debug:
+        msg: "##################################################"
+    - include_tasks: tasks/setup_mock_wifi_wpa3_sae.yml
+    - block:
+        - name: "TEST: wireless connection with WPA3 Personal"
+          debug:
+            msg: "##################################################"
+        - import_role:
+            name: linux-system-roles.network
+          vars:
+            network_allow_restart: true
+            network_connections:
+              - name: "{{ interface }}"
+                state: up
```

Comment:
How about making this `state: down` on RHEL, so we can have a test there as well?

Reply:
sounds good.

Reply:
Updated. Now we can test on RHEL as well.